src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -3,3 +3,13 @@ View the documentation at https://wiki.computercraft.cc
 Visit the forum at https://forums.computercraft.cc
 You can disable these messages by running "set motd.enable false"
 You can create directories with "mkdir".
+Want to see hidden files? Run "set list.show_hidden true".
+Run "list","ls" or "dir to see all files in a directory.
+You can delete files and directories with "delete" or "rm".
+Use "pastebin put" to upload a program to pastebin.
+Use "pastebin get" to download a program from pastebin.
+Use "pastebin run" to run a program from pastebin without downloading it.
```

Comment:
Maybe "without saving it first."? Running a file from the internet without downloading it would be quite impressive!